Routes/users.js

```diff
@@ -1,34 +1,34 @@
-// import express from "express";
-// import { PrismaClient } from "@prisma/client";
+import express from "express";
+import { PrismaClient } from "@prisma/client";
 
-// const router = express.Router();
-// const prisma = new PrismaClient();
+const router = express.Router();
+const prisma = new PrismaClient();
 
-// router.get("/", async (req, res) => {
-//   const users = await prisma.user.findMany();
-//   res.json(users);
-// });
+router.get("/", async (req, res) => {
+  const users = await prisma.user.findMany();
+  res.json(users);
+});
 
-// router.post("/", async (req, res) => {
-//   const { email } = req.body;
-//   const newUser = await prisma.user.create({
-//     data: {
-//       email,
-//     },
-//   });
-//   res.json(newUser);
-// });
+router.post("/", async (req, res) => {
+  const { email } = req.body;
+  const newUser = await prisma.user.create({
+    data: {
+      email,
+    },
+  });
+  res.json(newUser);
+});
 
-// router.put("/:id", async (req, res) => {
-//   const { id } = req.params;
-//   const { email } = req.body;
-//   const updatedUser = await prisma.user.update({
-//     where: { id: parseInt(id) },
-//     data: {
-//       email,
-//     },
-//   });
-//   res.json(updatedUser);
-// });
+router.put("/:id", async (req, res) => {
+  const { id } = req.params;
+  const { email } = req.body;
+  const updatedUser = await prisma.user.update({
+    where: { id: parseInt(id) },
```

Comment:
It's generally recommended to use the `Number()` instead of the parse functions.

1. Octal interpretation: When using parseInt(), if the string starts with a leading zero (e.g., "0123"), it will be interpreted as an octal number (base 8) instead of a decimal number. This can lead to unexpected results.

2. Hexadecimal interpretation: Similarly, if the string starts with "0x" or "0X", parseInt() will interpret it as a hexadecimal number (base 16).

3. Trailing characters: Both parseInt() and parseFloat() will ignore any trailing characters in the input string, which can lead to incorrect conversions. For example, "123abc" will be converted to 123 instead of throwing an error.

4. NaN handling: If the input string is not a valid number, parseInt() and parseFloat() will return NaN (Not a Number). However, this can be misleading, as NaN is a valid value in JavaScript, and it's not always clear whether the conversion failed or the input was genuinely invalid.

5. Lack of error handling: The parse functions do not provide any built-in error handling mechanisms. If the input string is invalid, they will simply return NaN or a incorrect value, without providing any information about the error.